packages/jscpd/__tests__/reporters.test.ts

```diff
@@ -2,9 +2,10 @@ import {expect} from 'chai';
 import {jscpd} from '../src';
 import {green, grey} from 'colors/safe';
 import sinon = require('sinon');
+import path = require('path');
 
 
-const pathToFixtures = __dirname + '/../../../fixtures';
+const pathToFixtures = path.join(__dirname + '/../../../fixtures');
```

Comment:
This is needed because we need to compare the full path generated by the xcode reporter.
By using `path.join` it interprets the `../` and generates the direct path.

Reply:
are we going to do that just in test?

Reply:
I did it here because the way the path to the test files was constructed. On the xcode reporter test the full path needs to match. The constructed path was `/rootpath/folder1/folder2/folder3/../../../fixtures/file.c` and ond the xcode reporter output it was `/rootpath/fixtures/file.c`. Both point top the same file but are different strings.

Ideally each time a path is manipulated the corresponding `path` function is used. I think we should move this in to a different pull request/issue.